### PR TITLE
Make it possible to be enable all `mdast-util-toc` options on TypeScript

### PR DESCRIPTION
**Pull request by babie**
Fix: #32

### CODE REVIEW

**Review by wooorm — APPROVED** [review]

File: types/remark-toc-tests.ts [hunk 1]
```diff
@@ -21,3 +21,11 @@ remark().use(toc, {
...
   skip: 'skip-heading'
 })
+
+remark().use(toc, {
+  prefix: 'prefix-'
+})
+
+remark().use(toc, {
+  parents: ['root', 'blockquote']
+})
```
Comment (wooorm) [inline types/remark-toc-tests.ts:31]: can you add a final enter here?

Comment (ChristianMurphy) [inline types/remark-toc-tests.ts:31]:
```suggestion
})

```

**Review by ChristianMurphy — APPROVED** [review]
Looks like this changed in https://github.com/remarkjs/remark-toc/commit/00c466f763eb5a7a451831a862a31f0003e57145
Thanks @babie!

File: types/remark-toc-tests.ts (hunk 1 above)
Comment (babie) [reply types/remark-toc-tests.ts]: ok, thx.

**Comment by wooorm** [comment]
Thanks @babie, [released](https://github.com/remarkjs/remark-toc/releases)!

**Pull request merged**